src/platform/apollolake/include/platform/lib/memory.h

```diff
@@ -275,12 +275,15 @@
 #define HEAP_RT_COUNT256		128
 #define HEAP_RT_COUNT512		8
 #define HEAP_RT_COUNT1024		4
+#define HEAP_RT_COUNT2048		0
+#define HEAP_RT_COUNT4096		1
```

Comment:
can you clarify why this is needed for ASRC?

Reply:
The ASRC algorithm core needs a pretty large dynamically allocated internal state variables storage of about 3kB. The size depends on actual conversion rates and PCM frame size. The other option would be to allocate this from buffers pool instead of but in the platforms I tested it the area ran out of allocated space.